client/blocks/app-banner/index.jsx

```diff
@@ -202,7 +180,9 @@ export class AppBanner extends Component {
 					statGroup="calypso_mobile_app_banner"
 					statName="impression"
 				/>
-				<img className="app-banner__illustration" src={ this.getBannerImage() } alt="" />
+				<div className={ classNames( 'app-banner__circle', 'is-top-left', 'is-yellow' ) } />
```

Comment:
I don't think it is necessery to use `classNames(...)` in this case. 
instead you can just use `className="app-banner__circle is-top-left is-yellow"`

Reply:
Oh, cool! That makes it a lot cleaner. Changed in 5c16b3d.